src/controllers/FontController.ts

```diff
@@ -38,6 +38,15 @@ export class FontController {
         return res.getFontById(fontId).then((result) => getEditorResponseData<DocumentFont>(result));
     };
 
+    /**
+     * This method returns the default font.
+     * @returns
+     */

```

Comment:
the lifetime is for the entire existence of the session right?
Maybe that's something to mention in the docs?